src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -2593,6 +2593,9 @@
   <data name="InvalidOperation_TimerAlreadyClosed" xml:space="preserve">
     <value>The Timer was already closed using an incompatible Dispose method.</value>
   </data>
+  <data name="InvalidOperation_TimeZoneLookupParsing" xml:space="preserve">
+    <value>Internal Error in parsing tzlookup file {1}.</value>
```

Comment:
We try to put quotes around paths. Generally "

Reply:
Done, thanks!